test/e2e/framework/providers/gce/gce.go

```diff
@@ -79,7 +83,10 @@ func factory() (framework.ProviderInterface, error) {
 		return nil, fmt.Errorf("Error building GCE/GKE provider: %v", err)
 	}
 
-	// Arbitrarily pick one of the zones we have nodes in
+	// Arbitrarily pick one of the zones we have nodes in, looking at prepopulated zones first.
```

Comment:
Do we want to validate the "Must include Zone" restriction in the comment above here?

Reply:
Yeah, doing so